README.md

```diff
@@ -17,6 +17,24 @@ text snippets.
 | -------- | -------------------------- | ------------------------------------------------ |
 | `GET`    | `/classify`                | Return subclass predictions for given text       |
 
+## How to run?
+
+### From command line
+
+1. Clone this repository
+1. Download required [assets](https://s3.amazonaws.com/pqai.s3/public/pqai-assets-latest.zip) and extract them to `/assets` directory
```

Comment:
It would be better to create a separate assets file, containing only those assets that are required by classifier.